src/index.html

```diff
@@ -1,12 +1,44 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="page__body">
+    <div class="page__calendar calendar calendar--start-day--mon calendar--month-length--30">
+      <div class="calendar__day calendar__day--1"></div>
```

Comment:
No need to write this additional class
```suggestion
      <div class="calendar__day"></div>
```